tools/contractdeployer/cli.go

```diff
@@ -6,6 +6,11 @@ import (
 	"strings"
 )
 
+const (
+	chainIDPlaceholder           = 0
```

Comment:
Little comment here might be good to say these are the go default values used for comparison to see if an argument value was passed.

Reply:
Added.